modules/installation-creating-gcp-lb.adoc

```diff
@@ -0,0 +1,141 @@
+
+.. Export the cluster network location:
++
+----
+$ export CLUSTER_NETWORK=`gcloud compute networks describe ${INFRA_ID}-network --format json | jq -r .selfLink`
+----
+
+.. Export the control plane subnet location:
+----
+$ export CONTROL_SUBNET=$(gcloud compute networks subnets describe <master-subnet> --region=${REGION} --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} --format json | jq -r .selfLink)
+----
+Where `<master-subnet>` is the name of the subnet in the shared VPC to deploy the control plane machines to.
+
+
+.. Export each zone that the cluster uses:
++
+----
+$ export ZONE_0=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[0] | cut -d "/" -f9`
+----
++
+----
+$ export ZONE_1=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[1] | cut -d "/" -f9`
+----
++
+----
+$ export ZONE_2=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[2] | cut -d "/" -f9`
+----
+
+. Create a `02_lb.yaml` resource definition file:
+** For an internal cluster, run the following command:
++
+----
+$ cat <<EOF >02_lb.yaml
+imports:
+- path: 02_lb_int.py
+
+resources:
+- name: cluster-lb-int
+  type: 02_lb_int.py
+  properties:
+    cluster_network: '${CLUSTER_NETWORK}' <1>
+    control_subnet: '${CONTROL_SUBNET}' <2>
+    infra_id: '${INFRA_ID}' <3>
+    region: '${REGION}' <4>
+    zones:
+    - '${ZONE_0}'
+    - '${ZONE_1}'
+    - '${ZONE_2}'
+EOF
+----
+<1> `cluster_network` is the `selfLink` URL to the cluster network.
+<2> `control_subnet` is the `selfLink` URL to the control subnet.
+<3> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
+<4> `region` is the region to deploy the cluster into, for example `us-east1`.
+
+** For an external cluster, run the following command:
++
+----
+$ cat <<EOF >02_lb.yaml
+imports:
+- path: 02_lb_ext.py
+- path: 02_lb_int.py
+resources:
+- name: cluster-lb-ext
+  type: 02_lb_ext.py
+  properties:
+    infra_id: '${INFRA_ID}' <1>
+    region: '${REGION}' <2>
+- name: cluster-lb-int
+  type: 02_lb_int.py
+  properties:
+    cluster_network: '${CLUSTER_NETWORK}' <3>
+    control_subnet: '${CONTROL_SUBNET}' <4>
+    infra_id: '${INFRA_ID}' <1>
+    region: '${REGION}' <2>
+    zones:
+    - '${ZONE_0}'
+    - '${ZONE_1}'
+    - '${ZONE_2}'
+EOF
+----
+<1> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
+<2> `region` is the region to deploy the cluster into, for example `us-east1`.
+<3> `cluster_network` is the `selfLink` URL to the cluster network.
+<4> `control_subnet` is the `selfLink` URL to the control subnet.
+
+. Create the deployment by using the `gcloud` CLI:
++
+----
+$ gcloud deployment-manager deployments create ${INFRA_ID}-lb --config 02_lb.yaml
+----
+
+. Export the cluster IP address:
++
+----
+$ export CLUSTER_IP=$(gcloud compute addresses describe ${INFRA_ID}-cluster-ip --region=${REGION} --format json | jq -r .address)
+----
+
+. Export the cluster public IP address:
```

Comment:
Again, Please keep an eye on the number, indentation, as well as the format.

Reply:
I think that the indentation is right now and that the problem is that it needs to say "For an external cluster, export..."